meetings/2023/LDM-2023-02-27.md

```diff
@@ -26,7 +26,7 @@ proposal generated significant discussion in the LDM, with roughly two ideologic
    should know when a method call has been intercepted: perhaps they need to indicate this by putting a sigil at the
    callsite, such as `#` (ie, `controller.MapGet#("/", CreateFeed)`).
 
-Not everyone fit into these precise camps, but these were the general sentiments. Some other ideas that were raised
+Not everyone fits into these precise camps, but these were the general sentiments. Some other ideas that were raised
```

Comment:
This is actually not a typo.

Reply:
Specifically, you'd use `fits` for present tense. This is past tense, so it's `fit`.